derex/runner/cli/__init__.py

```diff
@@ -43,12 +43,13 @@ def derex(ctx):
 
     click.echo(derex.get_help(ctx))
 
-    from .docker import get_exposed_container_names
+    from derex.runner.docker import get_exposed_container_names
 
-    containers = "\n".join(get_exposed_container_names())
-    click.echo(
-        f"\nThese containers are running and exposing an HTTP server on port 80:\n\n{containers}"
-    )
+    if get_exposed_container_names():
```

Comment:
We should invoke this function just once: it does a network call, so it's far from _free_.

Reply:
Nice catch ! Now we call `get_exposed_container_names` only once.